src/homepage/_posts/2019-08-30-latest-image-changes.markdown

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title:  "Apache James latest Docker images changes"
+date:   2019-08-30  09:09:47 +0100
+categories: james update
+---
+
+We have decided to change the latest Docker images behaviour.
+
+Since nowdays, such images were built on the `master` branch for each products (`linagora/james-memory`, `linagora/james-cassandra-rabbitmq-ldap-project`, ...).  
+This is not the way `latest` docker image should be, this [blog post](https://blog.container-solutions.com/docker-latest-confusion) is explaining this kind of misunderstood.  
+
+So we decided to follow the global Docker users behaviour which is to clone the _latest stable release_ as the Docker latest images.
+
+NB: to avoid such confusion, we strongly encourage you to not use the `latest` images for any of your production deployment.
```

Comment:
why, as it is the "latest stable release"?

Reply:
Because, the latest tag is not immutable.

It can change to another release, and your images might be silently upgraded...

Reply:
Agree, but you should explain that in the post.

Reply:
Then it will not be a post, but an article ;)

Nota Bene do not deserve extra explanations, IMO.

Reply:
I agree it can be easily rephrased as:
"NB: you should not use the `latest` image in a production deployment, otherwise you are at risk of jumping from one major release to another unexpectedly"